__tests__/app.test.js

```diff
@@ -51,6 +51,37 @@ afterAll(() => {
         });
     });
 
+    describe("GET /api/articles/:article_id", () => {
+        test("200: should respond with an article object, matching the inputted article ID", () => {
+            return request(app)
+            .get("/api/articles/5")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body).toBeInstanceOf(Object);
+                expect(body.article[0].article_id).toBe(5);
+            })
+        })
+        test("200: should respond with an article object containing author title, article_id, body, topic, created_at, votes, article_img_url", () => {
+            return request(app)
+            .get("/api/articles/6")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article[0]).toMatchObject({
```

Comment:
You can test with the actual data that article 6 has

Reply:
this test has been approved and merged in a previous ticket/PR. also, it's throwing errors if  change the test.